App.Loans/Interface/ILoanRepository.cs

```diff
@@ -1,12 +1,11 @@
-﻿using System.Collections.Generic;
+﻿using System.Linq;
 using App.Loans.Models;
 
 namespace App.Loans.Interface
 {
     public interface ILoanRepository
     {
-        IEnumerable<string> GetValues();
-        IEnumerable<Loan> GetLoans();
+        IQueryable<Loan> GetLoans();
```

Comment:
what is the reason for changing return type of the interface ?

Reply:
I'm changed back